src/stream/src/executor/hash_agg.rs

```diff
@@ -490,6 +498,11 @@ impl<K: HashKey, S: StateStore> HashAggExecutor<K, S> {
             input, mut extra, ..
         } = self;
 
+        let mut group_key_invert_idx: Vec<isize> = vec![-1; extra.input_schema.len()];
+        for (group_key_seq, group_key_idx) in extra.group_key_indices.iter().enumerate() {
+            group_key_invert_idx[*group_key_idx] = group_key_seq as isize;
```

Comment:
`group_key_invert_idx` can be built once on initialization

Reply:
> `group_key_invert_idx` can be built once on initialization

fixed